src/main/java/uk/gov/cshr/repository/InviteRepositoryMockImpl.java

```diff
@@ -65,6 +65,11 @@ public boolean existsByForEmailAndStatus(String email, InviteStatus status) {
         return false;
     }
 
+    @Override
```

Comment:
Can this class by removed?  As I added this when I first joined the project to try and mock the GovNotify.  A few months later I remember you saying you deleted all the spring profiles when you did a production release.  Therefore its not being used anymore.

Reply:
Class removed